src/shared/checkExistApiConfig.ts

```diff
@@ -18,6 +18,7 @@ export function checkExistKey(config: ApiConfiguration | undefined) {
 				config.vsCodeLmModelSelector,
 				config.requestyApiKey,
 				config.unboundApiKey,
+				config.firecrawlApiKey,
```

Comment:
Not sure we want to add this here - I think this is used to know when to show the welcome view

Reply:
Removed.